.github/workflows/regression-report.yml

```diff
@@ -391,7 +391,7 @@ jobs:
           if [[ "$DIFF_PROJECTS_LINK" == *.properties ]]; then
             cp "$DIFF_PROJECTS_LINK" .ci-temp/projects.properties
           else
-            cp "$DIFF_PROJECTS_LINK" .ci-temp/projects.yml
+            cp "$DIFF_PROJECTs_LINK" .ci-temp/projects.yml
```

Comment:
please review your own PR code changes ... it help a lot .

revert this change.